analyzers/src/SonarAnalyzer.VisualBasic/Facade/VisualBasicSyntaxFacade.cs

```diff
@@ -119,12 +106,15 @@ public override SyntaxNode RemoveConditionalAccess(SyntaxNode node)
         return whenNotNull;
     }
 
+    public override bool IsStatic(SyntaxNode node) =>
```

Comment:
This file has some noise of moving unchanged methods around. It would be good to have the same method ordering as on C#, without the shuffle